walletui.py

```diff
@@ -0,0 +1,103 @@
+import ecdsa
+import hashlib
+import requests
+from functools import reduce
+
+
+KEYSTORE = "keys.txt"
+
+
+def gen_key_pairs(n):
+    with open(KEYSTORE, "w") as keyfile:
+        for i in range(n):
+            sk = ecdsa.SigningKey.generate()
+            pk = sk.verifying_key
+            skhex = sk.to_string().hex()
+            pkhex = pk.to_string().hex()
+            keyfile.write(f"{pkhex}\n{skhex}\n\n")
+
+
+def main():
+    host = "http://127.0.0.1:8000/"
```

Comment:
Add it to the env file. Import it from there.

Reply:
this file (right now) acts as a 'frontend' for our project and host represents where our backend is...right now I have modified it to let the user set the host address in the environment instead of an env file as I think that is better from the perspective of end-user...hopefully in the future with a "real" frontend, we would not need this `walletui.py` file